simapp/params/proto.go

```diff
@@ -8,17 +8,16 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/tx"
 )
 
-// MakeEncodingConfig creates an EncodingConfig for an amino based test configuration.
+// MakeEncodingConfig creates an EncodingConfig for a non-amino based test configuration.
 func MakeEncodingConfig() EncodingConfig {
-	amino := codec.NewLegacyAmino()
+	cdc := codec.NewLegacyAmino()
```

Comment:
I prefer `amino` as the variable name here.

Reply:
`cdc` was the shorthand for Amino codec when there was only one codec. There are some leftovers where `cdc` is still used, but yes, we should use `amino/aminoCdc/legacyAmino`

Reply:
`cdc` is what we use almost everywhere. It has a proper type - if you hove on a cdc, when it's used you will see that it's an Amino codec. And we should use 2 at the same time. So if we change it will be easier in the future (less renames).